care/facility/models/post_covid_data.py

```diff
@@ -0,0 +1,206 @@
+    (e.value, e.name) for e in AnticoagulantModeOfTransmission
+]
+
+
+class PostCovidData(BaseModel):
+    consultation = models.OneToOneField(
+        PatientConsultation,
+        on_delete=models.PROTECT,
+        null=False
+    )
+    patient = models.ForeignKey(
+        "PatientRegistration",
+        on_delete=models.PROTECT,
+        null=True
+    )
+    date_of_onset_symptoms = models.DateField(
+        default=None,
+        null=True,
+        verbose_name="Date of first appearance of symptoms"
+    )
+    post_covid_time = models.IntegerField(
+        choices=PostCovidTimeChoices,
+        default=1,
+        blank=False
+    )
+    date_of_test_positive = models.DateField(
+        default=None,
+        null=False,
+        blank=False,
+        verbose_name="date when patient was tested positive"
+    )
+    date_of_test_negative = models.DateField(
+        default=None,
+        null=False,
+        blank=False,
+        verbose_name="date when patient was tested negative"
+    )
+
+    testing_centre = models.TextField(default="", null=True)
+    pre_covid_comorbidities = JSONField(
+        default=dict,
+        null=True,
+        verbose_name="diseases before coronavirus"
+    )
+    post_covid_comorbidities = JSONField(
+        default=dict,
+        null=True,
+        verbose_name="diseases after coronavirus"
+    )
+
+    treatment_facility = models.ManyToManyField(
+        "Facility",
+        null=True
+    )
+    treatment_duration = models.IntegerField(
+        default=None,
+        null=False,
+        blank=False,
+        verbose_name="Number of days for which treatment last"
+    )
+    covid_category = models.IntegerField(
+        choices=CovidCategoryChoices,
+        blank=False
+    )
+
+    vitals_at_admission = JSONField(default=dict, null=True)
+
+    condition_on_admission = models.TextField(default="", null=True, blank=True)
+    condition_on_discharge = models.TextField(default="", null=True, blank=True)
+    icu_admission = models.BooleanField(default=False, null=False, blank=False)
+    oxygen_requirement = models.BooleanField(default=False, null=False, blank=False)
+
+    oxygen_requirement_detail = models.IntegerField(
+        choices=OxygenTypeChoices, default=1
+    )
+
+    mechanical_ventilations_niv = models.IntegerField(default=0, null=True)
+    mechanical_ventilations_invasive = models.IntegerField(default=0, null=True)
+    antivirals = models.BooleanField(default=False, null=False)
+
+    antivirals_drugs = ArrayField(
+        models.CharField(max_length=100),
+        default=list,
+        null=True,
+        blank=True
+    )
+    steroids = models.BooleanField(default=False, null=True)
+    steroids_drugs = ArrayField(
+        JSONField(default=dict, null=False),
+        default=list,
+        null=True,
+        blank=True
+    )
+
+    anticoagulants = models.BooleanField(default=False, null=True)
+    anticoagulants_drugs = ArrayField(
```

Comment:
We could just use `JSONField(default=list)`, instead of using an ArrayField and a JSON field inside that.